starter-code/javascript/chronometer.js

```diff
@@ -1,35 +1,43 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
+Chronometer.prototype.startClick = function() {
+  var that = this;
+  this.intervalId = setInterval(function() {
+    that.currentTime++;
+  }, 1000);
+};
+
+Chronometer.prototype.setMinutes = function() {
+  return Math.floor(this.currentTime / 60);
+};
+
+Chronometer.prototype.setSeconds = function() {
+  return Math.floor(this.currentTime % 60);
+};
+
+Chronometer.prototype.twoDigitsNumber = function(value) {
+  if (value < 10) {
+    return "0" + value;
+  } else {
+    return "" + value;
+  }
+};
+
+Chronometer.prototype.setTime = function() {
+  this.startClick;
+
+  this.minutes = this.twoDigitsNumber(this.setMinutes);
+  this.seconds = this.twoDigitsNumber(this.setSeconds);
+};
 
 // Chronometer.prototype.setMilliseconds = function () {
 
 // };
 
-// Chronometer.prototype.stopClick = function () {
-  
-// };
+Chronometer.prototype.stopClick = function() {};
```

Comment:
This method should user clearInterval to clear the interval you started on startClick